### PR TITLE
Some minor changes and clean-up

### PR DESCRIPTION
**Pull request by blaumeise20**

### CODE REVIEW

File: DiscordBot.Modules/CommandModules/HelpModule.cs [hunk 1]
```diff
@@ -12,43 +13,49 @@ namespace DiscordBot.Modules.CommandModules
...
         {
-            _commands = commands;
             _map = map;
+            _commands = commands;
+            _configuration = configuration;
         }
 
         [Command("help")]
-        [Summary("Lists this bot's commands.")]
+        [Summary("Zeigt hilfe an.")]
```
Comment (SIMULATAN) [inline DiscordBot.Modules/CommandModules/HelpModule.cs:28]: Zeigt die Hilfe an.

File: DiscordBot.Modules/CommandModules/WeatherModule.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public async Task GetWeatherLegacyAsync([Remainder] string location)
             //Check if returned Values are valid
             if (WeatherStream == null)
             {
-                await Context.Channel.SendMessageAsync("Location invalid");
+                await Context.Channel.SendMessageAsync("Ort ungültig!");
```
Comment (SIMULATAN) [inline DiscordBot.Modules/CommandModules/WeatherModule.cs:30]: Ungültiger Ort!

File: DiscordBot.Modules/ReactionModules/ReactionTestModule.cs [hunk 3]
```diff
@@ -23,22 +23,22 @@ public override async Task<bool> ReactionAddedAsync()
                 _ => ""
             };
 
-            await ReplyAsync($"Reaction added! '{emote}'");
+            await ReplyAsync($"Reaction hinzugefügt! '{emote}'");
```
Comment (SIMULATAN) [inline DiscordBot.Modules/ReactionModules/ReactionTestModule.cs:26]: Reaktion

File: DiscordBot.Modules/ReactionModules/ReactionTestModule.cs [hunk 4]
```diff
@@ -23,22 +23,22 @@ public override async Task<bool> ReactionAddedAsync()
...
 
             return true;
         }
 
         public override async Task<bool> ReactionRemovedAsync()
         {
-            await ReplyAsync($"Reaction Removed! '{base.Context.Reaction.Emote.Name}'");
+            await ReplyAsync($"Reaction entfernt! '{base.Context.Reaction.Emote.Name}'");
```
Comment (SIMULATAN) [inline DiscordBot.Modules/ReactionModules/ReactionTestModule.cs:33]: ...

File: DiscordBot.Modules/ReactionModules/ReactionTestModule.cs [hunk 5]
```diff
@@ -23,22 +23,22 @@ public override async Task<bool> ReactionAddedAsync()
...
             // Application insights also works here:
             // throw new AggregateException("I don't like this");
             return true;
         }
 
         public override async Task<bool> ReactionClearedAsync()
         {
-            await ReplyAsync($"Reactions Cleared!");
+            await ReplyAsync($"Reactions geleert!");
```
Comment (SIMULATAN) [inline DiscordBot.Modules/ReactionModules/ReactionTestModule.cs:41]: ...
geleert -> gelöscht

File: DiscordBot/Services/Base/CommandHandlingService.cs [hunk 6]
```diff
@@ -94,7 +94,7 @@ ReactionType reactionType
                 {
                     if (!(scope.ServiceProvider.GetService(type) is ReactionModuleBase module))
                     {
-                        _logger.LogWarning($"Invalid type in {nameof(ReactionModuleRegistry)} found: '{type.FullName}'");
+                        _logger.LogWarning($"Ungültiger Type in {nameof(ReactionModuleRegistry)} gefunden: '{type.FullName}'.");
```
Comment (SIMULATAN) [inline DiscordBot/Services/Base/CommandHandlingService.cs:97]: Type -> Typ

File: DiscordBot/Services/Base/CommandHandlingService.cs [hunk 7]
```diff
@@ -133,12 +133,12 @@ private void UpdateMessagePrefix(string prefix)
 
             if (!_messagePrefix.HasValue)
             {
-                _logger.LogWarning($"Prefix has invalid length ({prefixLength}). Defaulting to '!'");
+                _logger.LogWarning($"Prefix hat eine ungültige Länge ({prefixLength}). '!' wird verwendet.");
```
Comment (SIMULATAN) [inline DiscordBot/Services/Base/CommandHandlingService.cs:136]: maybe another message?
Doesn't sound right to me...

Comment (blaumeise20) [reply DiscordBot/Services/Base/CommandHandlingService.cs:136]: I also thought about adding the ability to have a prefix with a length more than one :-)

File: DiscordBot.CoderDojoInfoModule/CoderDojoInfoModule.cs [hunk 8]
```diff
@@ -27,15 +25,15 @@ public class CoderDojoInfoModule : ModuleBase {
...
+                response.Append($"Hier die Termine für dich, {Context.User.Mention}: \n"); 
                 foreach (var appointment in appointments) {
                     response.Append($"> {appointment.Date.ToString("dddd, dd.MM.yyyy", new CultureInfo("de-DE"))}  \n");
                 }
 
                 await ReplyAsync(response.ToString());
             }
             catch (Exception ex) {
-                Logger.LogError(ex, "Failed to get Appointments");
+                Logger.LogError($"Die Termine konnte nicht eingeholt werden: {ex}");
```
Comment (JKamsker) [inline DiscordBot.CoderDojoInfoModule/CoderDojoInfoModule.cs:36]: Revert this one, the logger can get more useful information later when its needed out of the exception object.

File: DiscordBot.Modules/CommandModules/FileExplorerModule.cs [hunk 9]
```diff
@@ -28,8 +26,8 @@ public async Task FilesAsync([Remainder] string commandText)
             var filesFolder = configuration["FileStore:DataFolder"];
             if (!Directory.Exists(filesFolder))
             {
-                logger.LogError("FileStore:DataFolder does not refer to an existing directory. Did you configure it in appsettings.json?");
-                await ReplyAsync("Bot not properly configured. Check logs for details.");
+                logger.LogError("FileStore:DataFolder referenziert kein existierendes Verzeichnis. Hast du es in appsettings.json konfiguriert?");
```
Comment (JKamsker) [inline DiscordBot.Modules/CommandModules/FileExplorerModule.cs:29]: Why are u guys always germanizing everything? Thats ugly :D

File: DiscordBot.Modules/CommandModules/HelpModule.cs [hunk 10]
```diff
@@ -137,52 +128,53 @@ public void AddCommands(ModuleInfo module, EmbedBuilder builder)
 
         public void AddCommand(CommandInfo command, EmbedBuilder builder)
```
Comment (JKamsker) [inline DiscordBot.Modules/CommandModules/HelpModule.cs:129]: What made you doing this refactorization?

File: DiscordBot.Modules/CommandModules/HelpModule.cs [hunk 11]
```diff
@@ -137,52 +128,53 @@ public void AddCommands(ModuleInfo module, EmbedBuilder builder)
...
                 f.Value = $"{command.Summary}\n" +
                 (!string.IsNullOrEmpty(command.Remarks) ? $"({command.Remarks})\n" : "") +
                 (command.Aliases.Any() ? $"**Aliases:** {string.Join(", ", command.Aliases.Select(x => $"`{x}`"))}\n" : "") +
-                $"**Usage:** `{usage}`";
+                $"**Verwendung:** `{_configuration.Value.CommandPrefix}{usage}`";
             });
         }
 
-        public string GetAliases(CommandInfo command)
+        public List<string> GetParameters(CommandInfo command)
```
Comment (JKamsker) [inline DiscordBot.Modules/CommandModules/HelpModule.cs:147]: What made you doing this refactorization?

File: DiscordBot.Modules/CommandModules/HelpModule.cs [hunk 12]
```diff
@@ -137,52 +128,53 @@ public void AddCommands(ModuleInfo module, EmbedBuilder builder)
...
         {
-            var output = GetPrefix(command.Module);
-            output += $"{command.Aliases.FirstOrDefault()} ";
-            return output;
+            AddPrefix(command.Module, output);
+            output.Add(command.Aliases.First());
         }
 
-        public string GetPrefix(ModuleInfo module)
+        public void AddPrefix(ModuleInfo module, List<string> output)
```
Comment (JKamsker) [inline DiscordBot.Modules/CommandModules/HelpModule.cs:171]: What made you doing this refactorization?

File: DiscordBot.Modules/CommandModules/PingModule.cs [hunk 13]
```diff
@@ -1,23 +1,14 @@
 ﻿using System.Threading.Tasks;
 using Discord.Commands;
-using Microsoft.Extensions.Logging;
 
 namespace DiscordBot.Modules.CommandModules
 {
     public class PingModule : ModuleBase
     {
-        private readonly ILogger<PingModule> logger;
```
Comment (JKamsker) [inline DiscordBot.Modules/CommandModules/PingModule.cs:9]: Why did u remove the logger?

File: DiscordBot.Modules/CommandModules/FileExplorerModule.cs (hunk 9 above)
Comment (blaumeise20) [reply DiscordBot.Modules/CommandModules/FileExplorerModule.cs:29]: @SIMULATAN said, that you want that 😀

File: DiscordBot.Modules/CommandModules/HelpModule.cs (hunk 10 above)
Comment (blaumeise20) [reply DiscordBot.Modules/CommandModules/HelpModule.cs:129]: The Usage field is now fixed. You said that in our Discord, it doesn‘t look good.

File: DiscordBot.Modules/CommandModules/PingModule.cs (hunk 13 above)
Comment (blaumeise20) [reply DiscordBot.Modules/CommandModules/PingModule.cs:9]: Why do we use it in PingModule, when we don‘t use it in many other commands?

File: DiscordBot.Modules/CommandModules/FileExplorerModule.cs (hunk 9 above)
Comment (JKamsker) [reply DiscordBot.Modules/CommandModules/FileExplorerModule.cs:29]: Thats a lie.

File: DiscordBot.Modules/CommandModules/PingModule.cs (hunk 13 above)
Comment (JKamsker) [reply DiscordBot.Modules/CommandModules/PingModule.cs:9]: Logging is good practice. Not using it in other code is no valid justification for enforcing it in the whole codebase.

File: DiscordBot.Modules/CommandModules/HelpModule.cs (hunk 10 above)
Comment (JKamsker) [reply DiscordBot.Modules/CommandModules/HelpModule.cs:129]: What do you mean?

Comment (blaumeise20) [reply DiscordBot.Modules/CommandModules/HelpModule.cs:129]: I mean the usage

File: DiscordBot.CoderDojoInfoModule/CoderDojoInfoModule.cs [hunk 14]
```diff
@@ -27,15 +25,15 @@ public class CoderDojoInfoModule : ModuleBase {
                 StringBuilder response = new StringBuilder();
 
                 // the two blanks in front of linebreak are needed because discord uses Markdown
-                response.Append($"Hier die Termine für dich {Context.User.Username}:  \n"); 
+                response.Append($"{Context.User.Mention} Hier die Termine für dich:  \n");
```
[the remark this answers is not in the document]
Comment (JKamsker) [reply DiscordBot.CoderDojoInfoModule/CoderDojoInfoModule.cs]: Why do we germanize everything?

**Pull request merged**